### ISSUE TITLE
updating wiped my custom settings

### ISSUE THREAD

**Issue by DrMacabre** [Zyin055/Config-Presets#11]
hi,

I updated the extension today to find out it wiped all the settings i had carefully customized for my needs. That's less than convenient.

May be not updating the config.json when updating the script would be a good idea.

**Comment by DrMacabre** [comment]
It would be a great addition to the script to simply be able to save current settings to the config.json instead of editing it by hand.

**Comment by Zyin055** [comment]
This repo is the old version of this script and is now legacy. It's been updated to an extension which shouldn't wipe your config.json: https://github.com/Zyin055/Config-Presets

**Comment by DrMacabre** [comment]
ok thank you
So i submitted in the wrong repo because im using the extension from automatic 1111 and it still wiped my config.
![image](https://user-images.githubusercontent.com/49474452/208726176-1b1cc8ed-9017-499b-a48e-2ac3f207db4a.png)

**Comment by Zyin055** [comment]
I tried to figure out why this happened to you. It's programmed to only create a config.json file on a fresh install (or of it's missing/deleted).

I looked into the A1111 code and they do a `repo.git.reset('--hard', 'origin')` when updating extensions which is an aggressive way to update. I think what happened is this aggressive way to update deleted the config.json file because I had added it to .gitignore, so I hope this only happens once and never again.
